### ISSUE TITLE
cgroups controller: Make sure cgroup are in valid state on startup

### ISSUE THREAD

**Issue by jbreitbart** [lrr-tum/poncos#48]
We need to check the state of a cgroup before starting applications. They could exist and be in a frozen state. If that is a case poncos will not work as expected.

**Comment by jbreitbart** [comment]
@spickartz poncos can only send a request to migfra and ask for cleanup of all cgroups, i.e. delete all poncos cgroups. Is this supported?

**Comment by spickartz** [comment]
Yes, you can pass either the cgroup's name or a regex to delete all cgroups matching it.
